frb_codegen/src/others.rs

```diff
@@ -55,16 +62,30 @@ pub fn modify_dart_wire_content(content_raw: &str, dart_wire_class_name: &str) -
         ),
     );
 
-    content
+    content = content
         .replace("final class DartCObject extends ffi.Opaque {}", "")
         /*.replace(
             "class _Dart_Handle extends ffi.Opaque {}",
             "base class _Dart_Handle extends ffi.Opaque {}",
         )*/
-        .replace("typedef WireSyncReturn = ffi.Pointer<DartCObject>;", "")
+        .replace("typedef WireSyncReturn = ffi.Pointer<DartCObject>;", "");
+
+    // For ONLY regular configs: erase class block code which are shared.
+    // The redundant classes are due to the forward declaration in c header file for regular block.
+    // I didn't find a way to let it not generated in dart, so here remove it after dart code generated.
```

Comment:
nit: to avoid confusion from future readers

```suggestion
    // I (@dbsxdbsx) didn't find a way to let it not generated in dart, so here remove it after dart code generated.
```